.github/workflows/pytest.yml

```diff
@@ -0,0 +1,24 @@
+name: Run Python Tests with Pytest
+on:
+    push:
+        branches:
+            - mainline
+    pull_request:
+        branches:
+            - mainline
+
+jobs:
+    build:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
+            - name: Install Python 3
+              uses: actions/setup-python@v1
+              with:
+                  python-version: 3.9
+            - name: Install dependencies
+              run: |
+                  python -m pip install --upgrade pip
+                  pip install -r cdk/requirements.txt
+            - name: Test Lambdas
+              run: pytest cdk/visit/lambda_code
```

Comment:
Will this run recursively?

Reply:
Yes, it will hit each file that has the `import pytest` line inside of `cdk/visit/lambda_code`